docs/topics/ref_known-issues-7-3-2.adoc

```diff
@@ -11,3 +11,8 @@ This section provides highlighted known issues in {ProductFullName} version 7.3.
 .{ProductShortName} {WebName} connection breaks when you sort dependencies by labels after an analysis
 
 When you sort the dependencies by labels after analyzing an application in the {ProductShortName} {WebName}, the {WebName} displays an error about retrieving data. Currently, there is no workaround available. link:https://issues.redhat.com/browse/MTA-5981[(MTA-5981)]
+
+.CLI analysis performance in disconnected environments degrades with Maven search enabled
```

Comment:
The bug title was changed today to reflect more accurately the bug into. Note that this is not  a CLI specific bug:
"Analysis performance is significantly degraded in a disconnected environment or if maven central is not available and maven search is not disabled"
If this tile is considered too long, then I'll shorten it to:
"Analysis performance is significantly degraded in a disconnected environment or if maven central is not available"